gateway/apicast-0.1-0.rockspec

```diff
@@ -12,6 +12,7 @@ dependencies = {
   'liquid',
   'argparse',
   'penlight',
+  'sha1'
```

Comment:
There should be no need. There is [ngx.sha1_bin](https://github.com/openresty/lua-nginx-module/blob/master/README.markdown#ngxsha1_bin) and [`resty.sha1`](https://github.com/openresty/lua-resty-string#synopsis).